test/run-drun/heartbeat.mo

```diff
@@ -22,7 +22,7 @@ actor {
      while (count < max) {
        ignore await raw_rand(); // yield to scheduler
        attempts += 1;
-       if (attempts >= 50 and count == 0)
+       if (attempts >= 200 and count == 0)
```

Comment:
Yuck, that's precisely the kind of issues I was worried about with timer-based heartbeats. It's an IC simulation, we should be able to control the scheduling independent of walltime! (Also pinging @mraszyk)

Reply:
Yeah, a synthetic clock would be useful.